frontend/src/components/modules/Tournaments/Signup/Signup.tsx

```diff
@@ -161,7 +163,9 @@ const Signup: React.FC = (): ReactElement => {
           }}
           disabled={
             userId === undefined ||
-            (Boolean(tournament.passwordEnabled) && password === "")
+            (Boolean(tournament.passwordEnabled) &&
```

Comment:
Tip about clean code: in Boolean names it's advised to use 'is' , 'has' as a prefix.

https://training.galaxyproject.org/training-material/topics/admin/tutorials/terraform/tutorial.html

Reply:
Good effort as well man 👍🏻